File: Java/Loops/src/Divisibilityby8usingdowhile.java
/*
Write a program that displays all numbers divisible by 8 between 1 and 200 using do while
 */
public class Divisibilityby8usingdowhile {
    public static void main(String[] args){
        int i=0;
        do {
            if (i%8==0){
                System.out.print(i+", ");
            }

            i++;
        }
        while (i<=200);
    }
}

File: C++/Loops/Divisibility by 8 using do...while.cpp
/*Write a program that displays all numbers divisible by 8 between 1 and 200 using do…while loop*/
#include <iostream>
using namespace std;
int main()
{
    int number;
    number=0;
    do{
        if (number%8==0){
        cout<<number<<", ";}
        }
    while (++number<=200);
    return 0;
}
